.github/workflows/docs.yaml

```diff
@@ -86,12 +87,18 @@ jobs:
 
       - name: Get requirements
         run: |
-          make requirements-r
-          make requirements-py
+          task setup-r-tools
+          task setup-python-tools
 
       - name: Build docs
-        run: make docs
-        shell: bash
+        run: task build-website
+
+      - name: upload docs
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: actions/upload-artifact@v4
+        with:
+          name: docs
+          path: docs
```

Comment:
what does this do?

Reply:
Upload the website generated in PR as an artifact.
Download the artifact to check if the website is rendering correctly.
Without this, it is difficult to check whether your website was built correctly.